libraries/AP_Scripting/applets/VTOL-quicktune.lua

```diff
@@ -0,0 +1,387 @@
+--[[
+ fast tuning of VTOL gains for multirotors and quadplanes
+
+ This should be used in QLOITER mode for quadplanes and LOITER mode
+ for copters, although it will work in other VTOL modes
+
+--]]
+
+local PARAM_TABLE_KEY = 8
+local PARAM_TABLE_PREFIX = "QUIK_"
```

Comment:
QUIK could mean anything quick. I think perhaps QT for "Quick Tune" might be more appropriate

Reply:
i'll check with naming guru @rmackay9

Reply:
I'm not personally too concerned about the names for parameters in scripts... although I guess scripts are becoming more important.